release/src/milestones.ts

```diff
@@ -387,7 +388,7 @@ export async function checkMilestoneForRelease({
       issueNumber: issue.number,
       version,
       comment: 'Issue in milestone, cannot find commit',
-    });
+    }).catch((e) => console.error(`error adding issue ${issue.number} to project`, e));
```

Comment:
even if graphql fails, we can still post the info to slack, so we catch this error now.